qiskit_metal/_gui/main_window.py

```diff
@@ -114,6 +115,17 @@ def show_renderer_q3d(self):
 
     def delete_all_components(self):
         """Delete all components."""
+
+        # yesnoMsg = QMessageBox()
+        # yesnoMsg.setWindowTitle("Delete all components?");
+        # yesnoMsg.setText("Are you sure you want to clear all Metal components?");
```

Comment:
I believe this is a test you were playing with. Can you please remove it form the code in your next commit?
Cheers

Reply:
Sorry I committed again took and took that stuff out. I thought I had already cleaned up the code before committing it the first time, so I made some sort of mistake in the process.